ci/build_librepcb.sh

```diff
@@ -38,7 +38,18 @@ fi
 
 # build librepcb
 mkdir build && pushd build
-qmake ../librepcb.pro -r ${BUILDSPEC-} "QMAKE_CXX=$CXX" "QMAKE_CC=$CC" "QMAKE_CFLAGS=$CFLAGS" "QMAKE_CXXFLAGS=$CXXFLAGS" "PREFIX=`pwd`/install/opt"
+if [ "${UNBUNDLE-}" != "" ]; then
+  ADDITIONAL_ARGS="UNBUNDLE+=$UNBUNDLE"
+else
+  ADDITIONAL_ARGS=""
+fi
```

Comment:
Could you maybe move these lines which determine `ADDITIONAL_ARGS` to above the translations download step, i.e. below the lines which determine the other qmake arguments (`CFLAGS`, `BUILDSPEC`, ...)? :)